src/script/conversation/ConversationRepository.ts

```diff
@@ -1113,7 +1113,10 @@ export class ConversationRepository {
           action: async () => {
             try {
               const response = await this.conversation_service.postConversationJoin(key, code);
-              const conversationEntity = await this.getConversationById({domain: '', id: conversationId});
+              const conversationEntity = await this.getConversationById({
+                domain: this.userState.self().domain,
```

Comment:
Don't you have the real domain used at this point (since it's added to the url to join)? 

You could try to read the domain and fallback the the selfUser domain if not there. Would that work?

Reply:
ah true, i can do that

Reply:
this looks like it would require a change to the desktop repo also, currently only code and key are passed on

Reply:
Let's just plan for the future and try to read the `domain` param (with the fallback). 
In the future, when the desktop handles domain, the webapp will read it directly